contracts/governance/AxelarGatewayGovernance.sol

```diff
@@ -0,0 +1,87 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity 0.8.9;
+
+import { AxelarExecutable } from '@axelar-network/axelar-gmp-sdk-solidity/contracts/executable/AxelarExecutable.sol';
+import { IAxelarGovernance } from '../interfaces/IAxelarGovernance.sol';
+import { TimeLock } from '../util/TimeLock.sol';
+
+contract AxelarGatewayGovernance is AxelarExecutable, TimeLock, IAxelarGovernance {
+    enum Command {
+        ScheduleProposal,
+        CancelProposal
```

Comment:
```suggestion
        ScheduleUpgrade,
        CancelUpgrade
```

Reply:
not just for upgrades. governance can set rate limits too for e.g., so we should keep it generic